include/common/asm_macros_common.S

```diff
@@ -16,7 +16,7 @@
 	 * use a raw .align directive. Since func switches to a new section,
 	 * this would not have the desired effect.
 	 */
-	.macro func _name, _align=-1
+	.macro func _name, _align=2
```

Comment:
minor: it would be good to add a comment in line 15 to say the alignment defaults to 2 since both A32 and A64 instructions must be word aligned.